_emulator/pubsub-debug.log

```diff
@@ -0,0 +1,16 @@
+This is the Google Pub/Sub fake.
```

Comment:
Shouldn't we delete this?

Reply:
yeah not sure why this wasnt gitignored, will remove. thanks